src/libs/ReportUtils.ts

```diff
@@ -2983,7 +2983,7 @@ function getModifiedExpenseOriginalMessage(
         originalMessage.tag = transactionChanges?.tag;
     }
 
-    if ('taxAmount' in transactionChanges) {
+    if ('taxAmount' in transactionChanges && !('amount' in transactionChanges || 'currency' in transactionChanges || 'taxCode' in transactionChanges)) {
```

Comment:
NAB: but I think having the second condition as a seperately defined var would make this simpler to understand

Reply:
this doesn't match the other conditions in the function though... could we put the && condition on a newline maybe?

Reply:
I can add a comment for better understanding

Reply:
> NAB: but I think having the second condition as a seperately defined var would make this simpler to understand

NAB, comment is eventually resolved here https://github.com/Expensify/App/pull/42128/files#diff-577fcc5f3a5e4930916ff310e61c38edfa093792fe3ab55b9a30d2aecd8811dbR2985